src/streaming/text/TextSourceBuffer.js

```diff
@@ -263,12 +261,12 @@ function TextSourceBuffer(config) {
             return;
         }
 
-        if (mediaType === Constants.FRAGMENTED_TEXT) {
+        if (mediaInfo.codec.indexOf('application/mp4') !== -1) {
```

Comment:
can we use mediaInfo.isFragmented here?

Reply:
No, since `_appendFragmentedText()` does mp4 parsing, and we may have (even if not conformed to DASH IOP spec) fragmented text in VTT format.